service/framework.go

```diff
@@ -23,8 +23,8 @@ import (
 	"k8s.io/client-go/rest"
 
 	"github.com/giantswarm/flannel-operator/service/flannelconfig/v2/etcdv2"
-	"github.com/giantswarm/flannel-operator/service/flannelconfig/v2/resource/legacyv2"
-	"github.com/giantswarm/flannel-operator/service/flannelconfig/v2/resource/namespacev2"
+	"github.com/giantswarm/flannel-operator/service/flannelconfig/v2/resource/legacy"
+	"github.com/giantswarm/flannel-operator/service/flannelconfig/v2/resource/namespace"
 	"github.com/giantswarm/flannel-operator/service/flannelconfig/v2/resource/networkconfigv2"
```

Comment:
What with etcdv2 and networkconfigv2?

Reply:
Comes in the next PRs buddy. I wanted to split it down a bit.